test/loadfiles/cases/checker/CheckedRequests.d

```diff
@@ -224,9 +225,11 @@ class UnexpectedNotification
     {
         foreach (i, Field; typeof(U.tupleof))
         {
-            // U.tupleof[i].stringof is "(Aggregate).field". To get "field"
-            // strip the first "Aggregate".length + 3 characters.
-            const name = U.tupleof[i].stringof[U.stringof.length + 3 .. $];
+            version (D_Version2)
+                mixin("enum name = \""~ FieldName!(i, U) ~ "\";");
+            else
+                const istring name = FieldName!(i, U);
```

Comment:
version should not be needed

Reply:
See the commit comment. It needs to be `enum`, not in D2, `static immutable istring` causes a compile error.

Reply:
Ah, right, because of tuple foreach - sorry, missed commit body. In such cases I'd prefer to just use `FieldName!(i, U)` twice directly in the mixin - it is less confusing than the version in my opinion.